src/core/models/resultfactory.js

```diff
@@ -166,8 +167,9 @@ export default class ResultFactory {
    * @param {number} index
    * @param {number} distance
    * @returns {Result}
+   * @param {number} distanceFromFilter
```

Comment:
should probably go before the `@returns`

Reply:
fixed here: https://github.com/yext/answers/pull/1104/commits/8fe82adf244a8fdd6669772501b752fd5d17328a